sfn-to-sfn-cdk/lib/sfn-to-sfn-cdk-stack.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  aws_stepfunctions_tasks as tasks,
+  aws_stepfunctions as sfn,
+  aws_iam as iam,
+  Stack,
+  StackProps,
+  CfnOutput
+} from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+export class SfnToSfnCdkStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    // **********************
+    //  Child State machine 
+    // **********************
+    const wait = new sfn.Wait(this, 'WaitXSeconds', {
+      time: sfn.WaitTime.secondsPath('$.waitSeconds'),
+    });
+
+    const sendTaskTokenSuccess = new tasks.CallAwsService(this, 'SendTaskTokenSuccess', {
+      service: 'sfn',
+      action: 'sendTaskSuccess',
+      parameters: {
+        "TaskToken": sfn.JsonPath.stringAt('$.token'),
+        "Output": "{\"status\":\"Success\"}"
+      },
+      iamResources: ['arn:aws:states:' + Stack.of(this).region + ':' + Stack.of(this).account + ':stateMachine:*']
+    });
+
+    const childStateMachine = new sfn.StateMachine(this, 'ChildStateMachine', {
+      definition: wait.next(sendTaskTokenSuccess)
+    });
+
+    childStateMachine.role?.attachInlinePolicy(
+      new iam.Policy(this, 'sendTaskSuccessPermission', {
+        statements: [
+          new iam.PolicyStatement({
+            actions: ['states:SendTaskSuccess'],
+            resources: ['*'],
```

Comment:
@pubudusj maybe we should specify which resources here rather than everything? What you think?

Reply:
@boyney123 Yeah, agree. This is fixed.